tox.ini

```diff
@@ -1,6 +1,9 @@
 # For more information about tox, see https://tox.readthedocs.org/en/latest/
 [tox]
-envlist = py27,py34
+envlist = py{27,34,36}-pytest{325,330,latest}
 [testenv]
-deps=pytest       # install pytest in the venvs
+deps=
+    pytest325: pytest==3.2.5
+    pytest330: pytest==3.3.0
+    pytestlatest: pytest
```

Comment:
This looks great. Thank you.

Reply:
Hm. For some reason, this doesn't look like it's running the full matrix on Travis. Looking into it.

Reply:
Oh it's probably the `.travis.yml` file